--- Makefile   2021-03-12 04:48:21.268725458 +0100
+++ Makefile	2021-03-12 04:49:09.332724040 +0100
@@ -21,7 +21,7 @@
 #
 RMTOO_CALL ?= rmtoo
 RMTOO_CONFIG=Config.json
-RMTOO_CONTRIB_DIR ?= /usr/local/pkg/rmtoo/rmtoo
+RMTOO_CONTRIB_DIR ?= /usr/local/rmtoo
 
 CALL_RMTOO=${RMTOO_CALL} -j file://${RMTOO_CONFIG}
 
--- make.sh 2021-03-12 05:08:20.327933278 +0100
+++ make.sh	2021-03-12 05:11:23.154130460 +0100
@@ -1,3 +1,5 @@
 #!/bin/bash
-source /etc/profile.d/R50*
-make >make.log 2>&1
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+
+make > ${DIR}/make.log 2>&1
